### PR TITLE
tomb: init at version 2.2

### PR DESCRIPTION
**Pull request by ethanpailes**
###### Things done
- [x] Tested using sandboxing (`nix-build --option build-use-chroot true` or [nix.useChroot](http://nixos.org/nixos/manual/options.html#opt-nix.useChroot) on NixOS)
- Built on platform(s)
  - [x] NixOS
  - [ ] OS X
  - [ ] Linux
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Comment by ethanpailes** [comment]
I'm sorry for the staggering git incompetence displayed in my previous PR.

File: pkgs/os-specific/linux/tomb/default.nix [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    cp doc/tomb.1 $out/man.1
+
+    wrapProgram $out/bin/tomb \
+        --prefix PATH : "${pinentry}/bin" \
+        --prefix PATH : "${cryptsetup}/bin" \
+        --prefix PATH : "${gnupg1orig}/bin"
+  '';
+
+  meta = {
+    description = "File encryption on GNU/Linux.";
```
Comment (joachifm) [inline pkgs/os-specific/linux/tomb/default.nix]: Please remove the punctuation from the description, per the guidelines

File: pkgs/os-specific/linux/tomb/default.nix [hunk 2]
```diff
@@ -0,0 +1,41 @@
+{ stdenv, fetchurl, zsh, pinentry, cryptsetup, gnupg1orig, makeWrapper }:
+
+let
+    version = "2.2";
+in
+
+stdenv.mkDerivation rec {
```
Comment (joachifm) [inline pkgs/os-specific/linux/tomb/default.nix]: It makes no real difference, but the `rec` here is strictly unnecessary, as you're not making use of the recursion. Usually, `version` is defined within the attrset being passed to `mkDerivation`. `rec` is then used to allow referring to `version` within `name`.

**Comment by joachifm** [comment]
Looks good to me

**Comment by ethanpailes** [comment]
I have managed to squash these two commits together on my local branch, but I don't know how to overwrite the remote branch with the squashed commits. What is the best way to proceed?

File: pkgs/os-specific/linux/tomb/default.nix [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+    # manually patch the interpreter
+    sed -i -e "1s|.*|#!${zsh}/bin/zsh|g" tomb
+  '';
+
+  installPhase = ''
+    mkdir -p $out/bin
+    mkdir -p $out/man
+
+    cp tomb $out/bin/tomb
+    cp doc/tomb.1 $out/man.1
```
Comment (joachifm) [inline pkgs/os-specific/linux/tomb/default.nix]: Man pages go into `$out/share/man/man1`; in the current version the man page won't be available to the user.

**Comment by joachifm** [comment]
Apart from my last note, I think this is ready.

File: pkgs/os-specific/linux/tomb/default.nix [hunk 4]
```diff
@@ -24,7 +24,7 @@ stdenv.mkDerivation rec {
     mkdir -p $out/man
 
     cp tomb $out/bin/tomb
-    cp doc/tomb.1 $out/man.1
+    cp doc/tomb.1 $out/share/man/man1
```
Comment (joachifm) [inline pkgs/os-specific/linux/tomb/default.nix]: I suspect this will fail, the destination dir doesn't exist at this point.

**Comment by joachifm** [comment]
@ethanpailes please squash & let's merge this

**Comment by ethanpailes** [comment]
squashed

**Comment by joachifm** [comment]
Thank you

**Pull request merged**